openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1644,32 +1653,38 @@ The technology described in this specification was made available from contribut
 
-## Wallet Initiated Issuance during Presentation (Requires Presentation of Additional Credentials During Issuance) {#use-case-2}
+## Wallet Initiated Issuance during Presentation (Requires Presentation of Additional Credentials During Issuance) {#use-case-5}
 
 An End-User comes across a verifier app that is requesting the End-User to present a Credential, e.g., a university diploma. The Wallet determines the requested Credential type(s) from the presentation request and notifies the End-User that there is currently no matching Credential in the Wallet. The Wallet then offers the End-User a list of Credential Issuers, which might be based on a Credential Issuer list curated by the Wallet provider. The End-User picks the university she graduated from and is sent to that university's End-User experience (Web site or app).
 
 The End-User logs in to the university, which determines that the respective End-User account is not verified yet. Among multiple identification options, the End-User chooses to present a Credential from her Wallet. The End-User is sent back to the Wallet where she consents to present requested Credential(s) to the university. The End-User is sent back to the university End-User experience. Based on the presented Credential, the university completes the End-User verification, looks up End-User data in its database, and offers to issue a diploma as a Verifiable Credential. 
 
 Upon providing consent, the End-User is sent back to the Wallet. The Wallet informs the End-User Credential was successfully issued into the Wallet and is ready to be presented to the verifier app that originally requested presentation of that Credential.
 
+## Wallet Initiated Issuance after Installation {#use-case-6}
+
+The End-User installs a new government-provided Wallet. To get things started, the Wallet offers a selection of Credentials, that the the End-User may obtain, e.g. a national identity credential, a mobile driving license or a public transport ticket. The corresponding Credential Issuers (and their URLs) are pre-configured by the Wallet Provider. By clicking on one these options, the Issuance process starts and the End-User is directed to the Credential Issuer's website for authentication.
+
+Wallet Providers may also provide a market place where Issuers can register to be found for Wallet-initiated flows.
```

Comment:
```suggestion
```

I would say: out of scope for this specs as well

Reply:
its out-of-scope, but anything in the use cases is informative

Reply:
I don't have a strong opinion on that, so feel free to resolve this comment.
Just a warning: Wallet Providers are not defined term, we do not have any knowledge of this entity in this document